workflows/dev_deploy.yml

```
@@ -0,0 +1,66 @@
+name: CI/CD Pipeline
+
+on:
+  push:
+    branches: [ develop ]  # develop 브랜치에 push가 일어날 때 실행
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)